src/sources/community.js

```diff
@@ -26,7 +26,7 @@ class CommunitySource extends Source {
 
     this.defaultBranch = fetch(`https://api.github.com/repos/${this.owner}/${this.repoName}`, {headers: {authorization}})
       .then((res) => res.json())
-      .then((data) => data.default_branch)
+      .then((data) => data.default_branch ?? "master")
```

Comment:
I suspect it's always defined in a "good" response from GitHub's API, but if we get an error response and still successfully deserialize the body as JSON (since errors are reported as JSON), we'll have a structure in `data` that doesn't contain a `default_branch` property.